dags/wb_data.py

```diff
@@ -211,12 +225,21 @@ def transform_other(df: pd.DataFrame, non_countries: List[str], target_column: s
         # собираем общий файл с экономическими индикаторами
         df_vvp = transform_other(data['df_vvp'], data['non_countries'], 'vvp')
         df_population  = transform_other(data['df_population'], data['non_countries'], 'population')
+        df_electricity = transform_other(data['df_electricity'], data['non_countries'], 's1w1_electricity_access_percent')
```

Comment:
```suggestion
        df_electricity = transform_other(data['df_electricity'], data['non_countries'], 'electricity_access_percent')
```

Кажется так будет лучше 😏